source/guide_shopware-6.rst

```diff
@@ -36,6 +37,10 @@ All relevant legal information can be found here
 
   * https://github.com/shopware/platform/blob/master/LICENSE
 
+IMPORTANT
+=============
+Shopware v6.6.8.2 and newer require ~8.2.0 and higher. The currently highest Uberspace 7 php version (8.1.29) does not fulfill these system requirements. Installation and updates are possible via detours, but should be considered in productive environments.
```

Comment:
Require ~? Do you mean PHP? We got 8.1 - 8.4: https://manual.uberspace.de/lang-php/